src/compute/src/logging/compute.rs

```diff
@@ -242,26 +234,12 @@ pub fn construct<A: Allocate>(
                 Datum::MzTimestamp(datum.time),
             ])
         });
-        let peek_duration = peek_duration
-            .as_collection()
-            .arrange_core::<_, RowSpine<_, _, _, _>>(
-                Exchange::new(move |_| u64::cast_from(worker)),
-                "Arranged timely peek_duration",
-            )
```

Comment:
> This would remove all short-lived peeks before creating the Row.

It wouldn't because `peek_duration` is a monotonic collection. It contains a histogram for the durations of all peeks every seen, not just the currently active ones.

An arrangement here would still serve to merge together peeks that had the same duration and finished during the same logging interval the same time. I'm not sure how much of a benefit that would be.

Reply:
I ran this simple test on my laptop:

```
create cluster test replicas (r (size '4-4', introspection debugging true));
set cluster = test;

create view test2 as select generate_series as x from generate_series(1, 10);
create default index on test2;

-- repeat 10k times:
select count(*) from test2;
...
```

Without the arrangement it took 101 seconds, with the arrangement 99 seconds. Assuming that measurements collected on macOS are not too far removed from the reality on x86 Linux, we can expect some minor improvement, but probably not a significant one.